rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout, .sibbo-panel"],
+  "detectCmp": [{ "exists": "sibbo-cmp-layout" }],
+  "detectPopup": [{ "visible": ".sibbo-layout" }],
+  "optIn": [{ "click": "#acceptAllMain" }],
+  "optOut": [
+    { "click": ".sibbo-panel__aside__buttons a[data-nav=\"purposes\"]" },
+    { "click": ".sibbo-panel__main__header__actions a[data-focusable=\"reject-all\"]" },
+    { "wait": "500" },
+    { "click": ".sibbo-panel__main__footer__actions a[data-focusable=\"next\"]" },
```

Comment:
instead of unconditional wait, how about `.sibbo-panel__main__footer__actions a[data-focusable="next"]:not(.sibbo-cmp-button--disabled)`?

Reply:
As per our MM discussion, unconditional wait was replaced by some `waitFor` selectors for now.
In the future, we may want to switch sibbo into a class-based rules to better handle websites that use a iframe (I'll add a task to track that).